ml-package-versions.yml

```diff
@@ -80,18 +62,32 @@ tensorflow-2.x:
       pip install 'tf-nightly==2.5.0.dev20210112'
 
   models:
-    minimum: "2.0.0"
+    minimum: "1.15.4"
     maximum: "2.3.1"
+    requirements:
+      "< 2.2": ["h5py<3.0", "scikit-learn"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py --large
+      TF_MAJOR_VERSION=$(python -c "import tensorflow; print(tensorflow.__version__.split('.')[0])")
+
+      if [ "$TF_MAJOR_VERSION" == "1" ]; then
+        pytest tests/tensorflow/test_tensorflow_model_export.py --large
+      else
+        pytest tests/tensorflow/test_tensorflow2_model_export.py --large
+      fi
```

Comment:
Conditionally switch test scripts to run depending on the major version of tensorflow